src/neo/Network/P2P/Peer.cs

```diff
@@ -179,14 +180,14 @@ private void OnTcpConnected(IPEndPoint remote, IPEndPoint local)
             ImmutableInterlocked.Update(ref ConnectingPeers, p => p.Remove(remote));
             if (MaxConnections != -1 && ConnectedPeers.Count >= MaxConnections && !TrustedIpAddresses.Contains(remote.Address))
             {
-                Sender.Tell(Tcp.Abort.Instance);
+                TcpDisconnect(DisconnectReason.MaxConnectionReached);
```

Comment:
`DisconnectReason` should not appeared in the `Peer`.

Reply:
Fixed, I have moved this part of code into LocalNode.